### PR TITLE
[CI:DOCS] v4.2.0-rc3 release notes

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

#### Does this PR introduce a user-facing change?

```release-note
None
```

@TomSweeneyRedHat @rhatdan @baude @vrothberg @ashley-cui PTAL
/cc @containers/podman-maintainers

### CODE REVIEW

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Comment by lsm5** [comment]
needs rebase on https://github.com/containers/podman/pull/15194

**Comment by lsm5** [comment]
@ashley-cui PTAL. I included macos pkginstaller under the MISC section.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -40,6 +40,7 @@
 - The `podman stop` command now includes a new flag, `--filter`, to filter which containers will be stopped (e.g. `podman stop --all --filter label=COM.MY.APP`).
 - The Podman global option `--url` now has two aliases: `-H` and `--host`.
 - The `podman network create` command now supports a new option with the default `bridge` driver, `--opt isolate=`, which isolates the network by blocking any traffic from it to any other network with the `isolate` option enabled. This option is enabled by default for networks created using the Docker-compatible API.
+- The `podman image sign` command now supports creating a signature file that can be used later to verify the image.
```
Comment (vrothberg) [inline RELEASE_NOTES.md:43]: I don't see changes to `image sign`.  Is it meant to describe the sigstore changes?  Those went into `push`:

```suggestion
- Added the ability to create sigstore signatures in `podman push` and `podman manifest push`.
- Added an option to read image signing passphrase from a file.
```

**Comment by lsm5** [comment]
All changes addressed.

**Pull request merged**